File: 001/Erlang/problem001_tests.erl
-module(problem001_tests).

-include_lib("eunit/include/eunit.hrl").
-import(problem001, [sum35/1]).

sum35_test_() -> [
	?_assertEqual(0, sum35(0)),
	?_assertEqual(3, sum35(3)),
	?_assertEqual(8, sum35(5)),
	?_assertEqual(23, sum35(9))
	].

File: 001/Python/Problem001Tests.py
#!/usr/bin/env python

import unittest

import problem001

class Problem001Tests(unittest.TestCase):

    def testSum35(self):
        sum35 = problem001.sum35
        self.assertEqual(0, sum35(0))
        self.assertEqual(3, sum35(3))
        self.assertEqual(8, sum35(5))
        self.assertEqual(23, sum35(9))

unittest.main()
